Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -969,7 +969,7 @@ private static void Truncate(LiveResult result, DateTime start, DateTime stop)
 
         private string CreateKey(string suffix, string dateFormat = "yyyy-MM-dd")
         {
-            return $"{_job.DeployId}-{DateTime.UtcNow.ToString(dateFormat)}_{suffix}.json";
+            return $"{_job.DeployId}-{DateTime.UtcNow.ToStringInvariant(dateFormat)}_{suffix}.json";
```

Comment:
Should sanity check this doesn't change

Reply:
Confirmed via unit test: `Formattable_ToStringInvariant_UsesProvided_FormatParameter`